rio/src/RIO/Prelude/Logger.hs

```diff
@@ -204,6 +206,37 @@ logOther
   -> m ()
 logOther = logGeneric "" . LevelOther
 
+-- $withSource
+--
+-- There is a set of logging functions that take an extra 'LogSource'
+-- argument to provide context, typically detailing what part of an
+-- application the message comes from.
+--
+-- The built-in log functions as constructed with 'withLogFunc' and
```

Comment:
Actually, maybe the right thing to do instead is simply to start including the source when present. I don't think that in any way a meaningful breaking change. Thoughts? And would you be interested in amending this PR to include that?

Reply:
Sure! Will give it a shot.